packages/components/src/FilterBar/context/reducer/setupFiltersState.ts

```diff
@@ -0,0 +1,24 @@
+import { Filters } from "../../types"
+import { FiltersState, FiltersValues } from "../types"
+
+export const setupFiltersState = <ValuesMap extends FiltersValues>(
+  filters: Filters<ValuesMap>
+): FiltersState<ValuesMap> =>
+  filters.reduce<FiltersState<ValuesMap>>(
+    (acc, filter) => {
```

Comment:
Can we find a better name for `acc`?

Reply:
Lol, but it's an accumulator! I actually like it, since it's a temp variable which I associate with `reduce`.

That being said, what would you prefer it to be?